app/common/application/init.js

```diff
@@ -10,6 +10,7 @@
     .module('application', [
       'authentication',
       'bahmni.common.uiHelper',
+      'common.patient',
```

Comment:
I get the following error:
![image](https://user-images.githubusercontent.com/3810265/39393894-0a3d1bde-4acb-11e8-9cd3-eb9172ab3d2d.png)